tools/i18n-temp/ru.ts

```diff
@@ -1,60 +1,62 @@
 
   discarded: 'сброшена',
   'Discard a card': 'Сбросить карту',
 
-  'You Win!': 'You Win!',
-  'You Lose!': 'You Lose!',
-  'Tie Game': 'Tie Game',
+  'You Win!': 'Вы победили!',
+  'You Lose!': 'Вы проиграли!',
+  'Tie Game': 'Ничья!',
 
-  Preferences: 'Preferences',
-  'Your Name:': 'Your Name:',
-  "Opponent's Name:": "Opponent's Name:",
+  Preferences: 'Настройки',
+  'Your Name:': 'Ваше имя:',
+  "Opponent's Name:": "Имя противника:",
   'Choose a Tavern (Preset Preferences):':
-    'Choose a Tavern (Preset Preferences):',
-  Default: 'Default',
-  Customized: 'Customized',
-  'Starting Conditions': 'Starting Conditions',
-  'Victory Conditions': 'Victory Conditions',
-  'Other Preferences': 'Other Preferences',
-  'Cards in Hand': 'Cards in Hand',
-  'AI Type': 'AI Type',
-  'Apply & New Game': 'Apply & New Game',
-  Cancel: 'Cancel',
-
-  Language: 'Language',
-  Volume: 'Volume',
-  Mute: 'Mute',
-  Help: 'Help',
-  'Toggle Full Screen': 'Toggle Full Screen',
-
-  ERATHIAN: 'Use Erathian [%s] (for latin letters only)',
-
-  'ArcoMage HD': 'ArcoMage HD',
+    'Выберите Таверну (набор настроек):',
+  Default: 'По умолчанию',
+  Customized: 'Пользовательская',
+  'Starting Conditions': 'Стартовые условия',
+  'Victory Conditions': 'Условия победы',
+  'Other Preferences': 'Другие настройки',
+  'Cards in Hand': 'Карт в руке',
+  'AI Type': 'Тип ИИ', // couldn't find ingame
+  'Apply & New Game': 'Применить и начать игру', // 7 symbols longer, may want to use just "Начать игру!" instead if the button doesn't stretch
+  Cancel: 'Отмена',
+
+  Language: 'Язык',
+  Volume: 'Громкость',
+  Mute: 'Заглушить',
+  Help: 'Помощь',
+  'Toggle Full Screen': 'Полноэкранный режим',
+
+  ERATHIAN: 'Использовать Эрафийский [%s] (только для латиницы)', //probably not needed in RU locale at all
+
+  'ArcoMage HD': 'Аркомаг HD', // "Аркомаг" name is well known in Russia in its RU version, since just like Gwent (known in Russia as Гвинт), it was first introduced as a minigame inside another game, so they had to translate the name  
 
   DESC:
-    "Web-based open source HD clone of 3DO and NWC's 2000 card game Arcomage",
+    "опенсорсная веб-версия игры Arcomage, выпущенной 3DO и New World Computing в 2000 году", //name already mentions it's HD, and "Arcomage" fits perfectly here in english
 
   'Please go to %s to view more information, star the repo and follow %s there.':
-    'Please go to %s to view more information, star the repo and follow %s there.',
+    'Чтобы узнать больше, посетите %s, добавьте репозиторий в избранное и подпишитесь на %s.',
 
-  'the GitHub project page': 'the GitHub project page',
+  'the GitHub project page': 'страницу проекта на GitHub',
 
   'Below is a brief presentation of the rules:':
-    'Below is a brief presentation of the rules:',
+    'Правила игры:', // because full is too long here "Ниже вы можете найти краткое изложение правил:", and pretty pointless too, might even want to shorten it in source lang
```

Comment:
I'll change the English text. Actually I might change all these to an image or even an [interactive tour guide](https://reactour.js.org/).